db/migrate/20200524074928_create_service_resource_attachments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateServiceResourceAttachments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :service_resource_attachments do |t|
+      t.references :storage_service, :type => :bigint, :index => true
+      t.references :storage_resource, :type => :bigint, :index => true
+      t.boolean :compliant
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
```

Comment:
Also, an ems_ref on a join table doesn't make sense, are these actually returned by the provider API as its own resource?

Reply:
Yea, the attachments between the services and the resources are synced from the provider. 
The idea is that the provider can adjust attachments automatically based on rules.
@ereztheodorou, could you elaborate please?

Reply:
I have elaborated in the prev comment